src/js/Content/Features/Store/App/FWaitlistDropdown.js

```diff
@@ -5,67 +5,56 @@ import {Page} from "../../Page";
 export default class FWaitlistDropdown extends Feature {
 
     checkPrerequisites() {
-        return document.querySelector("#add_to_wishlist_area")
-            && SyncedStorage.get("add_to_waitlist")
-            && Background.action("itad.isconnected");
+        return SyncedStorage.get("add_to_waitlist")
+            && Background.action("itad.isconnected")
+            && document.querySelector("#add_to_wishlist_area") !== null;
```

Comment:
This basically checks if the app is owned or not now, maybe our `isOwned()` can be replaced with this.

Reply:
The first condition is not related to ownership, the second only works when you're connected to ITAD and the third is already included in `isOwned`.
I think we should distinguish between "owned" (Steam) and "collected" (ITAD)